src/schema/leaderboard.ts

```diff
@@ -100,6 +101,15 @@ export const typeDefs = /* GraphQL */ `
   }
 `;
 
+const excludedUsers = [
+  ghostUser.id,
```

Comment:
iirc there's also `system` user

Reply:
we jave `isSpecialUser` helper for ghost, system and so on